.travis.yml

```diff
@@ -78,7 +92,6 @@ jobs:
       - cd ..
       - bash docker_push.sh
       - cd ..
-      - ./trigger_policy_build.sh
```

Comment:
Is this removed on purpose?

Reply:
Yes and no. It should be in its own commit, which I will fix